ecep_cms/elasticsearch.yml

```diff
@@ -0,0 +1,14 @@
+#Use this to configure elasticsearch
```

Comment:
This config is unneeded because the ES container can be dropped.

Reply:
Indeed. I only need this as necessary.

I see now. J.Y. or J.R. created `apcd-cms` with customization opportunities we may not use. And I've just been copying them over.

_I cross another bridge of clarity today. Eventually, I cross enough to reach enlightenment._

Reply:
I will drop these _en masse_ in a follow up PR.

Reply:
And I will change it in `ecep_cms` here and test.